frontend/src/app/application-forms/fields/organization-name.component.html

```diff
@@ -1,5 +1,6 @@
 <ng-container [formGroup]="applicantInfo">
   <label id="organization-name-label" class="usa-input">{{ name }} <span class="required-fields-asterisk">*</span></label>
+  <p id="organization-name-hint-text" class="help-text usa-form-hint">NOTE: The {{ name }} field only allows 30 characters for input.</p>
```

Comment:
@Dmac26 per our conversation off github, even though it's a small addition we should check with @aQuib and get his approval for any changes that the user will see.

What'cha think @aQuib, I'm guessing this is g2g?